preki_funcutils/functions.py

```diff
@@ -104,6 +102,8 @@ def wrapper(event, context, *args, **kwargs):
         headers = event.get('headers', {})
         origin = headers.get('origin', headers.get('Origin', ''))
         stage = event.get('requestContext', {}).get('stage', 'dev')
+        allowed_origin = event.get(
+            'requestContext', {}).get('authorizer', {}).get('allowed_url', None)
```

Comment:
A very small and dumb change, but can we have the `allowed_url` parameter in camel case?. For consistency with the rest of the `requestContext` object

Consequently change it in this [PR](https://github.com/GoPreki/PrekiAPI/pull/631/files)